code/modules/mob/living/living_emote.dm

```diff
@@ -98,7 +99,10 @@
 		var/mob/living/carbon/human/H = user
 		if(H.dna.species)
 			message = H.dna.species.death_message
-			return pick(H.dna.species.death_sounds)
+			if(H.is_muzzled())
+				return
+			else
+				return pick(H.dna.species.death_sounds)
```

Comment:
```suggestion
			return pick(H.dna.species.death_sounds)
```
else is uneeded

Reply:
Removed this section so marking as resolved.